Lib/multiprocessing/context.py

```diff
@@ -258,7 +258,10 @@ def get_all_start_methods(self):
             return ['spawn']
         else:
             if reduction.HAVE_SEND_HANDLE:
-                return ['fork', 'spawn', 'forkserver']
+                if sys.platform == 'darwin':
```

Comment:
Are you certain `reduction.HAVE_SEND_HANDLE` will always be true on macOS? If not 100% this is true on older and future versions, then a similar change should be done in the `else` clause below.